episodes/01-introduction.md

```diff
@@ -65,8 +65,7 @@ would take up time even more time. A 'ready-to-go' server lets us focus on learn
 
 ## How to access the remote server
 
-You can log-in to the remote server using the instructions
-[here](https://datacarpentry.org/cloud-genomics/02-logging-onto-cloud#logging-onto-a-cloud-instance).
+You can log-in to the remote server using the [logging instructions from the Cloud Genomics lesson](https://datacarpentry.org/cloud-genomics/02-logging-onto-cloud#logging-onto-a-cloud-instance).
```

Comment:
```suggestion
You can log-in to the remote server using the [instructions from the Introduction to Cloud Computing for Genomics lesson](https://datacarpentry.org/cloud-genomics/02-logging-onto-cloud#logging-onto-a-cloud-instance).
```

I think 'logging' should be 'log-in', or removed. Also use the full name of the lesson